ghost/core/core/server/services/collections/service.js

```diff
@@ -1,25 +1,26 @@
 const {
-    CollectionsService,
-    CollectionsRepositoryInMemory
+    CollectionsService
 } = require('@tryghost/collections');
+const BookshelfCollectionsRepository = require('./BookshelfCollectionsRepository');
 const labs = require('../../../shared/labs');
 
 class CollectionsServiceWrapper {
     /** @type {CollectionsService} */
     api;
 
     constructor() {
-        const postsRepository = require('./PostsRepository').getInstance();
-        const collectionsRepositoryInMemory = new CollectionsRepositoryInMemory();
         const DomainEvents = require('@tryghost/domain-events');
+        const postsRepository = require('./PostsRepository').getInstance();
+        const models = require('../../models');
+        const collectionsRepositoryInMemory = new BookshelfCollectionsRepository(models.Collection);
```

Comment:
`bookshelfCollectionsRepository` for a variable name now? or just a `collectionsRepository` 😛

Reply:
Ha yeah 😝